### ISSUE TITLE
Some Unicode characters break terminal

### ISSUE THREAD

**Issue by hoffi** [zeit/hyper#589]
Some unicode chars break the terminal (e.g. ¶, “, ¢, ¡ or ∞)

I have attached a sample of the problem, where i insert ¶ (`Alt + 3` on a german mac keyboard) and some “ (`Alt + 2`).

![Issue](https://cloud.githubusercontent.com/assets/260453/17481340/f6668ca4-5d7c-11e6-9cac-6e01ecbd9d48.gif)

**Comment by dj-hedgehog** [comment]
I can't reproduce this with a German Mac keyboard on either ZSH, Fish and Bash. Can you tell us more about your config?

**Comment by hoffi** [comment]
```js
module.exports = {
  config: {
    // default font size in pixels for all tabs
    fontSize: 21,

    // font family with optional fallbacks
    fontFamily: '"Fantasque Sans Mono", "DejaVu Sans Mono", "Lucida Console", monospace',

    // terminal cursor background color (hex)
    cursorColor: '#F81CE5',

    // color of the text
    foregroundColor: '#fff',

    // terminal background color
    backgroundColor: '#000',

    // border color (window, tabs)
    borderColor: '#333',

    // custom css to embed in the main window
    css: '',

    // custom css to embed in the terminal window
    termCSS: '',

    // custom padding (css format, i.e.: `top right bottom left`)
    padding: '10px 8px',

    // some color overrides. see http://bit.ly/29k1iU2 for
    // the full list
    colors: [
    ]
  },

  // a list of plugins to fetch and install from npm
  // format: [@org/]project[#version]
  // examples:
  //   `hyperpower`
  //   `@company/project`
  //   `project#1.0.1`
  plugins: [
    'hyperline',
    'hyperterm-gruvbox-dark',
    'hyperterm-open-devtools',
  ],

  // in development, you can create a directory under
  // `~/.hyperterm_plugins/local/` and include it here
  // to load it and avoid it being `npm install`ed
  localPlugins: []
};
```

however this does also happen when i remove all plugins and use a system font.

I don't have this problem in iTerm2 with my same fish config.

**Comment by hoffi** [comment]
Another strange behavior is when i type "¶", then backspace, insert any text e.g. "test" and hit enter i get this:
`fish: Unknown command '�test'`

It seems like in this case the char is not completely deleted and this question mark is left over but is not visible in the prompt. I actually have to hit backspace twice to completely remove it.

**Comment by hoffi** [comment]
Fixed in 0.8.1
